<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WelcomeWindowDialog</class>
 <widget class="QWidget" name="WelcomeWindowDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Helvetica</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#WelcomeWindowDialog {
	background-image: url(&quot;:/WelcomWindowDialog/dialog-bacground-img.jpeg&quot;);
}
QWidget{
	background-color: transparent;
	color: white;
	border: 0 solid black;
}
QPushButton {
	border: 0 solid black;
	color: rgb(235, 247, 255);
	padding: 1em 2em 1em 2em;
	
}
QPushButton:hover {
	background-color: rgba(255,255,255, 32);
}
QPushButton:pressed {
	background-color: rgba(255, 255, 255, 128);
}
#exitButton {
	width: 36;
	height: 36;
	padding: 0;
	border-radius: 18;
}
#exitButton:hover {
	background-color: rgba(192, 0, 0, 32);
}
#exitButton:pressed {
	background-color: rgba(192, 0, 0, 128);
}
QGroupBox {
	background-color: #AA000000;
}

QComboBox {
	background-color: #AA000000;
	color: white;
}
#pathWidget {
	background-color: #AA000000;
}
#pathWidget QLineEdit {
	background-color: #AA666666;
}
#pathWidget QPushButton {
	padding: 2
}

</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="QPushButton" name="exitButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>36</width>
       <height>36</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>36</width>
       <height>36</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="WelcomeWindowDialogSources.qrc">
       <normaloff>:/WelcomWindowDialog/close-img.png</normaloff>:/WelcomWindowDialog/close-img.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QStackedWidget" name="centralStackedWidget">
     <property name="currentIndex">
      <number>2</number>
     </property>
     <widget class="QWidget" name="welcomPage">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="labelProjects">
         <property name="font">
          <font>
           <family>Helvetica</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>YOUR PROJECTS</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
         <property name="margin">
          <number>11</number>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QListWidget" name="projectList">
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>300</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Helvetica</family>
           <pointsize>12</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item row="0" column="1" rowspan="2">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QPushButton" name="createProjectButton">
           <property name="font">
            <font>
             <family>Helvetica</family>
             <pointsize>12</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>CREATE NEW PROJECT</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="openProjectButton">
           <property name="font">
            <font>
             <family>Helvetica</family>
             <pointsize>12</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>OPEN PROJECT</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="aboutProgrammButton">
           <property name="font">
            <font>
             <family>Helvetica</family>
             <pointsize>12</pointsize>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>ABOUT PROGRAMM</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="aboutProgrammPage">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QTextBrowser" name="textBrowser">
         <property name="html">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;justify&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:14pt; font-weight:600; color:#ffffff;&quot;&gt;Shader designer&lt;/span&gt;&lt;span style=&quot; font-size:14pt; color:#ffffff;&quot;&gt; - программа для разработки шейдерных программ, с использованием интерактивного визуального подхода к программированию.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="projectSettingsPage">
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Target language(will be possible change)</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Render method</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QRadioButton" name="radioForwardShading">
            <property name="text">
             <string>Forward Shading</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioDefferedShading">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="font">
             <font>
              <strikeout>true</strikeout>
             </font>
            </property>
            <property name="text">
             <string>Deferred Shading</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioLightPrePassShading">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="font">
             <font>
              <strikeout>true</strikeout>
             </font>
            </property>
            <property name="text">
             <string>Light Pre-Pass</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Render options</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="0" column="0">
           <widget class="QCheckBox" name="checkBox">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="font">
             <font>
              <strikeout>true</strikeout>
             </font>
            </property>
            <property name="text">
             <string>Use SSAO</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QCheckBox" name="checkBox_3">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="font">
             <font>
              <strikeout>true</strikeout>
             </font>
            </property>
            <property name="text">
             <string>Use PBR</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QCheckBox" name="checkBox_2">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="font">
             <font>
              <strikeout>true</strikeout>
             </font>
            </property>
            <property name="text">
             <string>Use shadows</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QCheckBox" name="checkBox_5">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="font">
             <font>
              <strikeout>true</strikeout>
             </font>
            </property>
            <property name="text">
             <string>CheckBox</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="comboBoxCompiller">
         <property name="font">
          <font>
           <family>Helvetica</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="editable">
          <bool>false</bool>
         </property>
         <property name="currentText">
          <string/>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QWidget" name="pathWidget" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>50</height>
          </size>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="1" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Solution name</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Solution folder path</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="editPath">
            <property name="font">
             <font>
              <family>Helvetica</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QPushButton" name="selectSolutionPathButton">
            <property name="text">
             <string>Select...</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="editName">
            <property name="font">
             <font>
              <family>Helvetica</family>
              <pointsize>10</pointsize>
             </font>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QCheckBox" name="checkBoxCreateFolder">
            <property name="text">
             <string>Create folder</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="3">
           <widget class="QPushButton" name="createNewSolutionButton">
            <property name="text">
             <string>Create</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <pointsize>24</pointsize>
      </font>
     </property>
     <property name="text">
      <string>SHADER DESIGNER</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QWidget" name="labelWidget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="backToMainButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="WelcomeWindowDialogSources.qrc">
          <normaloff>:/WelcomWindowDialog/back-img.png</normaloff>:/WelcomWindowDialog/back-img.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="pageNameLabel">
        <property name="font">
         <font>
          <family>Helvetica</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="WelcomeWindowDialogSources.qrc"/>
 </resources>
 <connections/>
</ui>
